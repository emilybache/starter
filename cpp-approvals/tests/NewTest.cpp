#include "ApprovalTests.hpp"
#include "catch.hpp"

using namespace ApprovalTests;

// All these tests should pass on your system.
// Please note that you will need a Diff Tool installed:
//  https://github.com/approvals/ApprovalTests.cpp/blob/master/doc/Reporters.md#supported-diff-tools

TEST_CASE("ApprovingText")
{
    Approvals::verify("Approval Tests can verify text via the golder master method ");
}

TEST_CASE("ApprovingWithReporters")
{
    // you can alter this to make it launch
    Approvals::verify("Reporters launch only on failure, by default they are the DiffReporter", DiffReporter());
}

TEST_CASE("Approval Is just an assertion library, you don't need to use it.")
{
    REQUIRE(std::string("No Approvals, only Catch") == "No Approvals, only Catch");
}
