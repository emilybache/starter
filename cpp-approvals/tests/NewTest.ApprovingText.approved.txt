Approval Tests can verify text via the golder master method 
